src/lib/components/event/event-metadata-expanded.svelte

```diff
@@ -0,0 +1,37 @@
+<script lang="ts">
+  import Badge from '$lib/holocene/badge.svelte';
+  import type { Payload } from '$lib/types';
+
+  import MetadataDecoder from './metadata-decoder.svelte';
+
+  export let value: Payload;
+</script>
+
+<div class="flex {$$props.class}">
+  <div class="content code-block-row">
+    <div class="flex flex-col">
+      <p class="text-sm">Summary</p>
+      <MetadataDecoder {value} fallback="" let:decodedValue>
```

Comment:
I think `fallback=""` is unnecessary here because the default value for that prop is already an empty string.